simple/cq_data.c

```diff
@@ -481,7 +481,7 @@ int main(int argc, char **argv)
 
 	hints->ep_attr->type = FI_EP_MSG;
 	hints->caps = FI_MSG;
-	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR;
+	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR | FI_RX_CQ_DATA;
```

Comment:
Support for CQ data should be specified using the domain attributes.  The code already does this correctly.  FI_RX_CQ_DATA is not a defined mode or cap.